test/AspNetCoreModule.TestSites.Standard/AspNetCoreModule.TestSites.Standard.csproj

```diff
@@ -1,21 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
-    <RuntimeFrameworkVersion>1.0.5</RuntimeFrameworkVersion>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
```

Comment:
@jkotalik I tried to build with this file change and ran into an build issue. So, I think we should postpone this file change after I figure out the build issue. FYI, I installed dotnetcore 2.0 SDK on top of VS 2017 but it did not solve the problem.